### PR TITLE
docs: updates to the help

### PR DESCRIPTION
**Pull request by ggarabedian**
Unified and improved the commands help.

Related to #4057

### CODE REVIEW

File: docs/man_pages/project/testing/debug-ios.md [hunk 1]
```diff
@@ -2,8 +2,18 @@
...
 
+### Description
+
+Initiates a debugging session for your project on a connected iOS device or in the iOS simulator. When necessary, the command will prepare, build, deploy and launch the app before starting the debug session. While debugging, the output from the application is printed in the console and any changes made to your code are synchronizes with the deployed app. <% if(isHtml) { %>Any debugging traffic is forwarded to port 8080 (or the next available one) from the device to the local machine.<% } %>
+
+<% if(isConsole && (isWindows || isLinux)) { %>WARNING: You can run this command only on macOS systems. To view the complete help for this command, run `$ tns help debug ios`<% } %>
+<% if((isConsole && isMacOS) || isHtml) { %>
+<% if(isHtml) { %>> <% } %>IMPORTANT: Before building for iOS device, verify that you have configured a valid pair of certificate and provisioning profile on your macOS system. <% if(isHtml) { %>For more information, see the [Code Signing](https://developer.apple.com/support/code-signing/) and [Maintain Signing Assets](https://help.apple.com/xcode/mac/current/#/dev3a05256b8) sections from the Apple Developer documentation.<% } %>
+
+### Commands
```
Comment (Fatme) [inline docs/man_pages/project/testing/debug-ios.md]:
```
Deploy on device/simulator, run the app, start Safari Web Inspector and attache the debugger.
``` 
is incorrect. Should be something like that
```
Deploy on device/simulator, run the app and attach the debugger to Chrome Developer Tools.
```

Comment (ggarabedian) [reply docs/man_pages/project/testing/debug-ios.md]: Does `Deploy on device/emulator, run the app and generate a Chrome DevTools link for debugging` sounds okay to you? Or perhaps `Deploy on device/emulator, run the app, attach a debugger and generate a Chrome DevTools link for debugging` sounds more correct (but longer)?

**Comment by DimitarTachev** [comment]
> @DimitarTachev, I have added the `hmr` flag to the `run android/ios`, `preview` and `debug android/ios` commands. Is the text "`--hmr` - Enables the hot module replacement (HMR) feature. HMR depends on `webpack` and requires the `--bundle` flag.`" correct?

@ggarabedian, when you set the `--hmr` flag, it will use `--bundle` out of the box. In other words, it requires webpack, but you don't need to set both the `--hmr` and `--bundle` flags. `--hmr` and `--bundle --hmr` are equivalent.

File: docs/man_pages/project/testing/debug-ios.md (hunk 1 above)
Comment (Fatme) [reply docs/man_pages/project/testing/debug-ios.md]: I'm voting for the second option 
```
Deploy on device/emulator, run the app, attach a debugger and generate a Chrome DevTools link for debugging.
```

**Pull request merged**